services/api/src/routes/auth.js

```diff
@@ -4,14 +4,14 @@ const validate = require('../utils/middleware/validate');
 const { authenticate } = require('../utils/middleware/authenticate');
 const tokens = require('../utils/tokens');
 const { sendWelcome, sendResetPassword, sendResetPasswordUnknown } = require('../utils/emails');
-const { BadRequestError, UnauthorizedError } = require('../utils/errors');
+const { BadRequestError } = require('../utils/errors');
 const { User, Invite } = require('../models');
 
 const router = new Router();
 
 const passwordField = Joi.string()
-  .min(6)
-  .message('Your password must be at least 6 characters long. Please try another.');
+  .min(12)
+  .message('Your password must be at least 12 characters long. Please try another.');
```

Comment:
can we make it 8 ? `at least 8 characters including a number and a lowercase letter` 
gonna be annoying to do 12, specially as you are building it

Reply:
I mean my local dev passwords are all `1111` anyway... it's a question of holding down the key 500ms longer :)
anyway lets discuss I'm fine with whatever we agree with

Reply:
just thinking of the default admin password, but that will bypass all this stuff i guess. So perhaps it doesn't matter after all.